main.rb

```diff
@@ -0,0 +1,165 @@
+require_relative 'planet'
+require_relative 'solar_system'
+require 'pry'
+require 'colorize'
+
+# WAVE 1
+# def main
+#
+#   earth = Planet.new('Earth', 'blue-green', 5.972e24, 1.496e8, 'Only planet known to support life')
+#
+#   puts earth.summary
+# end
+#
+# main
+
+# # WAVE 2
+# # Creating a solar system
+# solar_system = SolarSystem.new('Sol')
+#
+# # Creating planet earth
+# earth = Planet.new('Earth', 'blue-green', 5.972e24, 1.496e8, 'Only planet known to support life')
+# # Adding planet earth to the list
+# solar_system.add_planets(earth)
+#
+# # creating planet mars
+# mars = Planet.new('Mars', 'red', 6.420e23, 2.279e8, 'It is the second smallest planet')
+#
+# # adding planet mars to the list
+# solar_system.add_planets(mars)
+#
+# # Creating planent jupiter
+# jupiter = Planet.new('Jupiter', 'pearl', 1.898e27, 7.785e8, 'The largest planet in the solar system')
+#
+# # Adding planet jupiter to the list
+# solar_system.add_planets(jupiter)
+# list = solar_system.list_planets
+# puts list
+#
+# found_planet = solar_system.find_by_name('jupiter')
+#
+#
+# puts found_planet.summary
+# distance_earth_to_mars = solar_system.distance_between('earth', 'mars')
+#
+# puts distance_earth_to_mars
+
+# WAVE 3 Putting all the methods from wave 1 and wave 2
+def main
+  # Creating a solar systeme
+  solar_system = SolarSystem.new('Sun')
+
+  # Creating planet earth
+  earth = Planet.new('Earth', 'blue-green', 5.972e24, 1.496e8, 'Only planet known to support life')
+  # Adding planet earth to the list
+  solar_system.add_planets(earth)
+
+  # creating planet mars
+  mars = Planet.new('Mars', 'red', 6.420e23, 2.279e8, 'It is the second smallest planet')
+
+  # adding planet mars to the list
+  solar_system.add_planets(mars)
+
+  # Creating planent jupiter
+  jupiter = Planet.new('Jupiter', 'pearl', 1.898e27, 7.785e8, 'The largest planet in the solar system')
+
+  # Adding planet jupiter to the list
+  solar_system.add_planets(jupiter)
+
+  message_to_user(solar_system)
+
+end
+
+# Method for prompting the user to chose an option
+def message_to_user(solar_system)
```

Comment:
I love this method! I think it is responsible for an appropriate amount of things. I might suggest naming it different from `message_to_user` though?